### DISCUSSION TITLE
Space for Summary in Routes

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#378] [Q&A] [answered]
There is space for summarization of the overall km and duration at the bottom of the screen :-)

![Screenshot_20231116-085942_Cruiser](https://github.com/devemux86/cruiser/assets/121371629/6e658335-8b90-4809-ae5e-f0d7c14bb193)

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom thanks for the suggestion.

These are favorite routes. For statistics of navigation please see:
- https://github.com/devemux86/cruiser/discussions/342

**Reply by DolomitesTom** [reply]
Yes, but instead of summary of height meter, I would prefer the average km per daytrip in the summary line.
